### DISCUSSION TITLE
Discussion topics for Mar 14, 2022 Engineering Sync

### DISCUSSION THREAD

**Discussion by carlosfelix2** [CDCgov/prime-reportstream#4756] [Show and tell]
1. @jimduff-usds logger fatail errors
2. @jimduff-usds Encourage use of labels on Tickets and PRs that will then get picked up by the Change Log tool.

**Comment by bgantick** [comment]
On the horizon.. API working group formation. Purpose: a forum to go over proposed API changes and future direction/implementations.
